components/automate-cli/pkg/diagnostics/integration/auth_policies.go

```diff
@@ -20,7 +20,7 @@ type authPoliciesSave struct {
 func CreateAuthPoliciesDiagnostic() diagnostics.Diagnostic {
 	return diagnostics.Diagnostic{
 		Name: "auth-policies",
-		Tags: diagnostics.Tags{"auth"},
+		Tags: diagnostics.Tags{"auth", "iam-v1"},
```

Comment:
Oof shouldn't we have a test where we *do* migrate the v1 policies? 😩

Reply:
This is not the v1-to-v2 migration, this is used to (among other things) test an actual upgrade of the A2 version installed.

Also, we've got "did these policies get migrated?" tests; but they only assert that a migrated policy is there, I don't think they assert that they're _working_. *However*, implicitly, I believe this is tested for admin tokens 🤔